src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/clients/ErrorNotificationClientTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.blobrouter.clients;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import feign.FeignException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.blobrouter.config.ContextInitializer;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.created;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.unauthorized;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@ActiveProfiles(ContextInitializer.PROFILE_WIREMOCK)
+@AutoConfigureWireMock
+@ContextConfiguration(initializers = ContextInitializer.class)
+@SpringBootTest
+public class ErrorNotificationClientTest {
+
+    private static final ErrorNotificationRequest TEST_NOTIFICATION_REQUEST = new ErrorNotificationRequest(
+        "test_zip_file_name",
+        "test_po_box",
+        "test_error_code",
+        "test_error_description"
+    );
+
+    private static final ErrorNotificationResponse TEST_NOTIFICATION_RESPONSE =
+        new ErrorNotificationResponse("some id");
+
+    @Autowired
+    private ErrorNotificationClient client;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Test
+    public void should_return_Created_when_everything_is_ok_with_request() throws JsonProcessingException {
```

Comment:
should return notification id ?

Reply:
let me review test names - literally just copied without any changes